platform/web/tests/decode_with_json_schema_test.go

```diff
@@ -13,59 +14,83 @@ type decodeWithJSONSchemaTestData struct {
 	Body             string
 	ValidateResponse func(testProvider *testing.T, outcome error, data decodeWithJSONSchemaTestData)
 	JSONSchemaPath   string
+	DecoderOptions   web.DecoderOptions
+	ExpectedError    bool
```

Comment:
To me `ExpectedError` sounds like it'd be an error type, `ErrorExpected` or `ExpectsError` feels more like something that would be boolean imho